parm/diag_table.FV3_GFS_v17_p8

```diff
@@ -154,6 +154,8 @@
 "gfs_phys",  "BC_AOD_550",    "bc_aod550",     "fv3_history2d",      "all",  .false.,  "none",  2
 "gfs_phys",  "OC_AOD_550",    "oc_aod550",     "fv3_history2d",      "all",  .false.,  "none",  2
 "gfs_phys",  "SS_AOD_550",    "ss_aod550",     "fv3_history2d",      "all",  .false.,  "none",  2
+# Reflectivity from Thompson microphysics
```

Comment:
Shouldn't this be 'GFDL microphysics'?  We're in a GFS SDF.

Reply:
GFSv17 P8 is testing with Thompson!

Reply:
That's correct @michelleharrold. I'm not sure if it was different previously, but from what I can see currently the `FV3_GFS_v17_p8` suite uses Thompson microphysics: https://github.com/NOAA-EMC/fv3atm/blob/bba399053d3939241938f19ee598895eea54fd65/ccpp/suites/suite_FV3_GFS_v17_p8.xml#L76